### PR TITLE
Prepare Certificates for preview 8 release

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj [hunk 1]
```diff
@@ -3,18 +3,13 @@
...
-      - CertificateClient.RecoverDeletedCertificate has been renamed to CertificateClient.StartRecoverDeletedCertificate and now returns a RecoverDeletedCertificateOperation to track this long-running operation.
-      - CertificateIssuer.Administrators has been renamed to CertificateIssuer.AdministratorContacts.
-      - CertificateKeyType.Oct has been removed.
-      - The SubjectAlternativeNames class now allows you to add multiple types of subject alternative names using any of the DnsNames, Emails, and UserPrincipalNames collection properties.
-
-      See https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Certificates_4.0.0-preview.7/sdk/keyvault/Azure.Security.KeyVault.Certificates/ChangeLog.md for more detailed changes.
+      A small number of parameters were renamed or made required to better align with other languages and to reflect requirements of the Azure Key Vault service.
+
+      See https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Certificates_4.0.0-preview.8/sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md for more detailed changes.
```
Comment (annelo-msft) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj]: Does CHANGELOG need to be updated as part of this PR?

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj [hunk 2]
```diff
@@ -3,18 +3,13 @@
...
-      - CertificateClient.DeleteCertificate has been renamed to CertificateClient.StartDeleteCertificate and now returns a DeleteCertificateOperation to track this long-running operation.
-      - CertificateClient.RecoverDeletedCertificate has been renamed to CertificateClient.StartRecoverDeletedCertificate and now returns a RecoverDeletedCertificateOperation to track this long-running operation.
-      - CertificateIssuer.Administrators has been renamed to CertificateIssuer.AdministratorContacts.
-      - CertificateKeyType.Oct has been removed.
-      - The SubjectAlternativeNames class now allows you to add multiple types of subject alternative names using any of the DnsNames, Emails, and UserPrincipalNames collection properties.
-
-      See https://github.com/Azure/azure-sdk-for-net/tree/Azure.Security.KeyVault.Certificates_4.0.0-preview.7/sdk/keyvault/Azure.Security.KeyVault.Certificates/ChangeLog.md for more detailed changes.
+      A small number of parameters were renamed or made required to better align with other languages and to reflect requirements of the Azure Key Vault service.
```
Comment (annelo-msft) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj]: A few parameters?
I tend to put the verb first ... Renamed several parameters?
Would it be good to say SDK libraries in other languages for people who don't have full context on the SDK?
Separate out the Made required - was this to align with other languages or to reflect AKV requirements?  If one was one and one is the other, you could have two sentences with fewer clauses.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj (hunk 1 above)
Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj]: I keep it up to date as I make changes. It's ready.

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj (hunk 2 above)
Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Azure.Security.KeyVault.Certificates.csproj]: "Made required" - a bit of both. But mainly to align with the service. I'll keep it simple, though.

**Review by annelo-msft — APPROVED** [review]

**Review by schaabs — APPROVED** [review]

**Pull request merged**